docs/installation.rst

```diff
@@ -0,0 +1,386 @@
+
+Installing Anaconda Python
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We have chosen ``pyenv`` above because it is extremely lightweight and unobtrusive with your operative system. However, you might want to consider a more fully fledged Python distribution like Anaconda, specially if your work involved non-python packages or a lot of data science and machine learning tools.
+
+Regardless of the reason, if you want to follow this route just go to the official `Anaconda webpage <https://www.anaconda.com/>`_ and download and install a version appropriate for your operative system. Do not worry about the Python version as ``conda`` will let you choose that when creating a virtual environment.
+
+The installer should guide you step by step on the process of installing Anaconda and configuring your system to use it as your Python installation.
+
+Creating virtual environments
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Although not strictly necessary, **creating a virtual environment is highly recommended** regardless of how you installed Python. It will isolate users and developers from changes occurring on their operating system, and from conflicts between python packages and it ensures reproducibility from day to day.
+
+Using ``pipx`` ensures that putting each application it installs in its own virtual environment and running it under the hood. However, you can explicitly create and manage the virtual environment if you prefer.
+
+Creating a ``conda`` virtual environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This option is available only if you installed Anaconda Python. Depending on the settings you used when installing Anaconda and your operative system, you might have ``conda`` available in your normal terminal or you might need to use the Anaconda Prompt.
+
+``conda`` not only lets you create a virtual environment but also selecting which python version to use within, independently of the version of Anaconda Python installed, which means it can be an alternative to ``pyenv`` if it happens that you already have Anaconda installed in your system.
+
+To create an environment called ``muse_env`` run:
+
+.. code-block:: bash
+
+    conda create -n muse_env python=3.9
+
+Now, you can activate the environment with:
+
+.. code-block:: bash
+
+    conda activate muse_env
+
+Later, to recover the system-wide "normal" python, deactivate the environment with:
+
+.. code-block:: bash
+
+    conda deactivate
+
+Creating a virtual environment with ``venv``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Modern Python versions, regardless of their origin, come with a built in tool to create virtual environments, ``venv``. However, contrary to ``conda`` it does not let you select the version of Python that will be used - it will be the same one you are using to create the environment. Therefore, you still need to make sure your version of Python is compatible with MUSE.
+
```

Comment:
How do we check the python version in venv?

Reply:
Duplicated comment.